Corrections/Correction_scala.scala

```diff
@@ -0,0 +1,297 @@
+import java.util.{Random, Scanner}
+
+//
+//  ScalaCorrection.java
```

Comment:
Please correct the header, and rename your file to `Correction_Scala.scala` (uppercase).